### PR TITLE
NodeRemoving command for OV backend.

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- This PR contains part of the #1468 such as NodeRemovingCommand for OVModelTransformer

### Reason for changes

- Dividing into small PRs will allow us to have the desired functionality faster in the main branch

### Related tickets

- Part of functionality for 98621

### Tests

- Added test for the new command

### CODE REVIEW

File: nncf/experimental/openvino_native/graph/model_transformer.py [hunk 1]
```diff
@@ -107,3 +113,17 @@ def _insert_outputs(model: ov.Model, outputs: List[ov.Output]) -> ov.Model:
             extra_model_outputs.append(result)
 
         return ov.Model(model_outputs + extra_model_outputs, params)
+
+    def _apply_node_removing_transformation(self, transformations: List[OVNodeRemovingCommand]) -> None:
```
Comment (alexsu52) [inline nncf/experimental/openvino_native/graph/model_transformer.py:117]: Does the function support removing a node with multiple input ports?

Comment (KodiaqQ) [reply nncf/experimental/openvino_native/graph/model_transformer.py:117]: If you mean FakeQuantizes - yes, it removes it. 
The thing is that we need a quantization algorithm (aka MinMax) to get the quantized model and test the command correctly. And this is why the test removes the original FP nodes with one input now.

File: nncf/experimental/openvino_native/graph/model_transformer.py [hunk 2]
```diff
@@ -107,3 +113,17 @@ def _insert_outputs(model: ov.Model, outputs: List[ov.Output]) -> ov.Model:
...
+        """
+        Removes the layers from the model.
+        :param transformations: lisf of the node removing transformations.
+        """
+        for transformation in transformations:
+            node = self.name_to_node_mapping[transformation.target_point.target_node_name]
+
+            node_input = node.input(0).get_source_output()
```
Comment (l-bat) [inline nncf/experimental/openvino_native/graph/model_transformer.py:125]:
```suggestion
            node_input = node.input_value(0)
```

File: nncf/experimental/onnx/graph/transformations/commands.py [hunk 3]
```diff
@@ -105,7 +105,7 @@ def union(self, other: 'Command') -> 'Command':
         # Have a look at nncf/torch/graph/transformations/commands/PTInsertionCommand
         raise NotImplementedError()
 
-class ONNXNodeRemovingCommand(TransformationCommand):
+class ONNXQDQNodeRemovingCommand(TransformationCommand):
     """
     Removes nodes from the model.
```
Comment (kshpv) [inline nncf/experimental/onnx/graph/transformations/commands.py:110]: Please, update the description

**Review by kshpv — CHANGES_REQUESTED** [review]

**Review by l-bat — APPROVED** [review]

**Review by kshpv — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]

**Review by andrey-churkin — APPROVED** [review]

**Comment by andrey-churkin** [comment]
@KodiaqQ Could you please rebase your branch?

**Review by vshampor — APPROVED** [review]

**Pull request merged**